src/app/frontend/replicasetlist/replicasetlist.html

```diff
@@ -34,3 +34,9 @@
   </kd-content>
 </kd-content-card>
 <kd-zero-state ng-if="$ctrl.shouldShowZeroState()"></kd-zero-state>
+
+<kd-auto-refresh source="::$ctrl.replicaSetListResource"
```

Comment:
Another problematic scenario (here, I've no idea why it behaves so):

1. Open a new Dashboard window; go to the pod list.
2. Change the namespace to "kube-system".
3. After 10 seconds, the list of pods will change back to that for namespace "default".
   (However, the namespace selector will still show "kube-system").